usaspending_api/awards/tests/test_awards_v2.py

```diff
@@ -783,6 +806,18 @@ def test_award_psc_hierarchy_types(client, awards_and_transactions):
     }
 
 
+def test_special_characters(client, awards_and_transactions):
+    resp = client.get("/api/v2/awards/ASST_NON_:~$@*\"()%23/,^&+=`!'%/_. -_9700/")
+    assert resp.status_code == status.HTTP_200_OK
+
+    resp = client.get("/api/v2/awards/count/transaction/ASST_NON_:~$@*\"()%23/,^&+=`!'%/_. -_9700/")
+    assert resp.status_code == status.HTTP_200_OK
+    resp = client.get("/api/v2/awards/count/subaward/ASST_NON_:~$@*\"()%23/,^&+=`!'%/_. -_9700/")
+    assert resp.status_code == status.HTTP_200_OK
+    resp = client.get("/api/v2/awards/count/federal_account/ASST_NON_:~$@*\"()%23/,^&+=`!'%/_. -_9700/")
```

Comment:
```suggestion
    resp = client.get("/api/v2/awards/count/federal_account/ASST_NON_:~$@*\"()%23/,^&+=`!'%/_. Ǖ ǖ Ꞁ ¤⏠ ⏡ ⏦ ᶀ ᶁ⥔ ⥕¯\_(ツ)_/¯-_9700/")
```
If the idea is to take in any Unicode character, might as well get weird with it ¯\_(ツ)_/¯